.drone.yml

```diff
@@ -418,8 +418,10 @@ steps:
     when:
       cron: fail_security_scans
       event: cron
+      status:
+        - success
+        - failure
     depends_on:
-      - cron_clone_repos
```

Comment:
this needs to stay, as it relies on artefacts from the `hof-services-config` repo

`ALLOW_CVE_LIST_FILE: hof-services-config/infrastructure/trivy/.trivyignore.yaml`

Reply:
agreed to omit this, as the same dependency appears in the previous step `cron_build_image'.